editor.planx.uk/src/pages/Pay/MakePayment.tsx

```diff
@@ -170,14 +171,11 @@ export default function MakePayment({
     );
 
   const PaymentDetails = () => {
-    const $public = useStore((state) => state.$public);
     const [projectType, setProjectType] = useState<string | undefined>();
 
     useEffect(() => {
-      const fetchProjectType = async () => {
-        const projectType = await $public().formatRawProjectTypes(
-          rawProjectTypes,
-        );
+      const fetchProjectType = () => {
+        const projectType = formatRawProjectTypes(rawProjectTypes);
```

Comment:
nit: We have to use this slightly awkward syntax when calling an async function inside a `useEffect()` which required us to define the function in scope.

Now it's no longer async I think we can just simplify this by dropping the `useEffect()` and associated `useState()` - this isn't a reactive value, just a formatted prop.

```ts
const projectType = formatRawProjectTypes(rawProjectTypes);

// or if the above is a little slow/intensive (I don't really think so?)

const projectType = useMemo(formatRawProjectTypes(rawProjectTypes), [rawProjectTypes])
```

Reply:
Good catch ! Updated here `24ba79f` (no indication it needs a `useMemo`, but we can keep an eye on that in prod)